packages/xo-web/src/xo-app/hub/recipes/recipe.js

```diff
@@ -51,10 +54,33 @@ export default decorate([
           size: 'medium',
         })
 
-        const { masterName, nbNodes, network, sr, sshKey } = recipeParams
+        const {
+          staticIpAddress,
+          masterIpAddress,
+          networkMask,
+          gatewayIpAddress,
+          masterName,
+          nbNodes,
+          network,
+          sr,
+          sshKey,
+        } = recipeParams
+
+        let workerNodeIps
+        if (staticIpAddress === true) {
+          workerNodeIps = {}
+          for (let i = 0; i < nbNodes; i++) {
+            const key = 'workerIpAddress' + (i + 1).toString()
```

Comment:
Why not count from 1 in the loop? Also, the `.toString()` shouldn't be necessary.

Reply:
It starts at 0 because ```workerIpAddress``` was initially an array. When I changed it to a dictionary, I forgot to change the counting start to 1, which would have been more logical.

Anyway, I set it back to an array.